### PR TITLE
Remove gdal install requirement

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**

### CODE REVIEW

File: vignettes/04-synthesis-data.Rmd [hunk 1]
```diff
@@ -137,53 +136,21 @@ Below we retrieve all the available plots for a particular date, then find and c
...
 site.geom <- sites$geometry
 
-# Stripping out the extra information to get to the points
-complete_str <- str_match_all(site.geom, '(\\(\\(\\((.*)\\)\\)\\))')[[1]][, 3]
-bounds <- strsplit(complete_str, ', ')[[1]]
-
-# Getting the bounding box of the polygon
-bounding_box = getBounds(bounds)
+# Convert the polygon to something we can clip with. CRS value represents WGS84 Lat/Long
+site.shape = st_as_sfc(site.geom,crs = 4326)
```
Comment (dlebauer) [inline vignettes/04-synthesis-data.Rmd]: R style convention: use <- not = for assignment

File: vignettes/04-synthesis-data.Rmd [hunk 2]
```diff
@@ -221,61 +188,45 @@ image_files <-
...
-  # Remove the temporary file
-  if (file.exists(out_file)) 
-    file.remove(out_file)
-
+  
+  # If we can't get a greeness value we return zero
+  if (is.nan(greeness)) {
+    greeness = 0;
+  }
+
```
Comment (KristinaRiemer) [inline vignettes/04-synthesis-data.Rmd]: This should probably return an NA, not a 0. If it's zero, it looks like actual data (i.e., the red and green rasters are the same).

Comment (dlebauer) [reply vignettes/04-synthesis-data.Rmd]: For a demo can't we avoid inputs that return nan?

Comment (KristinaRiemer) [reply vignettes/04-synthesis-data.Rmd]: Two rasters currently return 0.

**Comment by KristinaRiemer** [comment]
Otherwise this looks good.

File: vignettes/04-synthesis-data.Rmd (hunk 1 above)
Comment (Chris-Schnaufer) [reply vignettes/04-synthesis-data.Rmd]: The text is in there. I've broken it out onto its own line to be more visible.

File: vignettes/04-synthesis-data.Rmd (hunk 2 above)
Comment (Chris-Schnaufer) [reply vignettes/04-synthesis-data.Rmd]: I've removed the two offending files.

**Pull request merged**